app/lib/actions/pulp3/orchestration/repository/update.rb

```diff
@@ -0,0 +1,18 @@
+module Actions
+  module Pulp3
+    module Orchestration
+      module Repository
+        class Update < Pulp3::Abstract
+          def plan(repository, smart_proxy)
+            sequence do
```

Comment:
we could probably do these concurrently, can't think of any reason why they need to be sequential